### PR TITLE
RFC-0322 Technical Editing

### PR DESCRIPTION
**Pull request by anselld**
## Description
Technical editing.

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [x] Documentation

## Checklist:

* [ ] I'm merging against the `development` branch
* [ ] I ran `cargo-fmt --all` before pushing
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by anselld** [comment]
Hi @hansieodendaal , please review my changes - thanks.

File: RFC/src/RFC-0322_VNRegistration.md [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 **Maintainer(s)**: [Cayle Sharrock](https://github.com/CjS77)
 
-# License
+# Licence
 
-[ The 3-Clause BSD License](https://opensource.org/licenses/BSD-3-Clause).
+[ The 3-Clause BSD Licence](https://opensource.org/licenses/BSD-3-Clause).
 
 Copyright 2019. The Tari Development Community
```
Comment (hansieodendaal) [inline RFC/src/RFC-0322_VNRegistration.md]:
```suggestion
Copyright 2019 The Tari Development Community
```

**Comment by hansieodendaal** [comment]
Hi @anselld, this is perfect. Please combine your commits.

**Comment by anselld** [comment]
Hi @hansieodendaal , I'm experiencing a problem combining the commits.

**Comment by anselld** [comment]
Hi @hansieodendaal , I've combined the PRs.

**Pull request merged**